internal/devpkg/package.go

```diff
@@ -238,6 +238,12 @@ func (p *Package) IsInstallable() bool {
 }
 
 func (p *Package) PatchGlibc() bool {
+	if p.CanonicalName() == "python" {
+		if p.patchGlibc != nil && !p.patchGlibc() {
```

Comment:
I think this actually still needs a tweak to distinguish between missing and false.

Reply:
Yeah, looks like function returns false in both cases https://github.com/jetify-com/devbox/blob/main/internal/devpkg/package.go#L113C1-L115C5

Reply:
https://github.com/jetify-com/devbox/pull/2251